Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -111,6 +111,35 @@ internal static void SeparateOculusFromMRTK()
                 "OK");
         }
 
+        /// <summary>
+        /// Initialize the Oculus Project Config with the appropriate settings to enable handtracking and keyboard support.
+        /// </summary>
+        [MenuItem("Mixed Reality Toolkit/Utilities/Oculus/Initialize Oculus Project Config")]
+        internal static void InitializeOculusProjectConfig()
+        {
+#if OCULUSINTEGRATION_PRESENT
+            // Updating the oculus project config to allow for handtracking and system keyboard usage
+            OVRProjectConfig defaultOculusProjectConfig = OVRProjectConfig.GetProjectConfig();
+            if (defaultOculusProjectConfig != null)
+            {
+                defaultOculusProjectConfig.handTrackingSupport = OVRProjectConfig.HandTrackingSupport.ControllersAndHands;
+                defaultOculusProjectConfig.requiresSystemKeyboard = true;
+
+                Debug.Log("Updated default project settings");
```

Comment:
Do we want this to be a bit more descriptive? Maybe something like "Enabled Oculus keyboard and hands/controllers" or something?

Reply:
Just missed this, will update message in a followup PR

Reply:
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8851 here we go.